requirements.txt

```diff
@@ -1,5 +1,5 @@
 # Everything needed in production
-werkzeug
+werkzeug<2.2.3
```

Comment:
Did you try <=2.2.3 by the way? I know I have 2.2.2, but..

Reply:
Yes I'm using it locally. We might not be using the specific problematic code in scout, but let's fix it!